src/ui/windows/TogglDesktop/TogglDesktop/ui/ViewModels/TimelineViewModel.cs

```diff
@@ -185,8 +185,9 @@ private static List<TimeEntryBlock> ConvertTimeEntriesToBlocks(List<Toggl.TogglT
                     HasTag = !entry.Tags.IsNullOrEmpty(),
                     IsBillable = entry.Billable,
                     Duration = entry.DateDuration,
-                    StartEndCaption = entry.StartTimeString + " - " + entry.EndTimeString
-                };
+                    StartEndCaption = entry.StartTimeString + " - " + entry.EndTimeString,
+                    IsResizable = height >= 14 //Resize handles take 5 px each (10 in total)

```

Comment:
At first I tried to handle this in xaml only, binding visibility to height and using a coverter. But height changes on drag event, and when it becomes less than 14 while user is dragging, the handles would suddenly disappear. I wanted to avoid this behavior.

Reply:
You can add an immutable Height property instead (call it e.g. HeightBeforeResizing, or call it Height and the other one HeightMutable/HeightResizable). Then handling it in XAML is easy.
It might be slightly more code, but if we continue to hardcode px in the viewmodel, changing the UI may become really hard.

Reply:
If you're concerned about that it's hardcoded in ViewModel, not in view, then I think the right way to go will be to create a *.cs file which would contain all the constants (and ScaleModes dictionary can also go there). And both views and viewModels would reference that constans.

Reply:
Decoupling, yay 😄 Sounds like a good idea 👍

Reply:
Can do it in this PR then 😄